clang/test/CMakeLists.txt

```diff
@@ -89,6 +89,7 @@ list(APPEND CLANG_TEST_DEPS
 if(CLANG_ENABLE_CIR)
   list(APPEND CLANG_TEST_DEPS
     cir-opt
+    cir-translate
```

Comment:
Thanks! I missed that. I can see that the incubator has `mlir-translate` as a dependency, too. Should we add this now as well?

Reply:
That won't be needed until we upstream one of the tests that uses it. Those are all in the Lowering/ThroughMLIR directory, which we won't be adding for a while.